src/main/java/seedu/address/storage/JsonSerializableAddressBook.java

```diff
@@ -11,24 +11,24 @@
 import seedu.address.commons.exceptions.IllegalValueException;
 import seedu.address.model.AddressBook;
 import seedu.address.model.ReadOnlyAddressBook;
-import seedu.address.model.person.Person;
+import seedu.address.model.client.Client;
 
 /**
  * An Immutable AddressBook that is serializable to JSON format.
  */
 @JsonRootName(value = "addressbook")
 class JsonSerializableAddressBook {
 
-    public static final String MESSAGE_DUPLICATE_PERSON = "Persons list contains duplicate person(s).";
+    public static final String MESSAGE_DUPLICATE_CLIENT = "Clients list contains duplicate Client(s).";
 
-    private final List<JsonAdaptedPerson> persons = new ArrayList<>();
+    private final List<JsonAdaptedClient> Clients = new ArrayList<>();
```

Comment:
should Clients be clients?

Reply:
for this, there are some errors when I change it to clients, not really sure how to solve them, it somehow fails 2 test cases but it passes the checkstyle test on Github, regarding the other checkstyles, I'll amend them accordingly thanks!

Reply:
ohh, if the checkstyle version/file in both GitHub and IntelliJ are already configured properly but the outputs are different, I'm not sure which one to follow

Reply:
to fix this issue (change Clients to client), refer to the new comment